src/components/new-safe/create/logic/index.ts

```diff
@@ -65,9 +71,13 @@ export const computeNewSafeAddress = async (
       saltNonce: props.saltNonce,
       safeVersion: safeVersion ?? getLatestSafeVersion(chain),
     },
+    isL1SafeSingleton: true,
   })
 }
 
+export const SAFE_TO_L2_SETUP_ADDRESS = '0x80E0d1577aD3d982BF2F49aAB00BfA161AA763c4'
```

Comment:
Is it already clear where this address will be stored and if it will be the same on every chain?

Reply:
Might be better to store it in constants.ts? 
I assume it will be the same on every chain but @schmanu may be better able to say